helm_deploy/hmpps-education-and-work-plan-api/values.yaml

```diff
@@ -33,6 +35,8 @@ generic-service:
   namespace_secrets:
     hmpps-education-and-work-plan-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+    PRISON_API_CLIENT_ID: "PRISON_API_CLIENT_ID"
```

Comment:
Ah - these 2 need to be indented, so they are children of `hmpps-education-and-work-plan-api`
(thats why you have a linting error)

Reply:
Opps! Good spot. Fixed.